diff --git a/libmicroros.mk b/libmicroros.mk
index 38c9b6a..4b70692 100644
--- a/libmicroros.mk
+++ b/libmicroros.mk
@@ -70,7 +70,7 @@ $(EXTENSIONS_DIR)/micro_ros_src/src:
 	git clone -b humble https://gitlab.com/ros-tracing/ros2_tracing src/ros2_tracing; \
 	git clone -b humble https://github.com/micro-ROS/micro_ros_utilities src/micro_ros_utilities; \
     touch src/rosidl/rosidl_typesupport_introspection_cpp/COLCON_IGNORE; \
-    touch src/rcl_logging/rcl_logging_log4cxx/COLCON_IGNORE; \
+    test ! -d src/rcl_logging/rcl_logging_log4cxx || touch src/rcl_logging/rcl_logging_log4cxx/COLCON_IGNORE; \
     touch src/rcl_logging/rcl_logging_spdlog/COLCON_IGNORE; \
     touch src/rclc/rclc_examples/COLCON_IGNORE; \
 	touch src/rcl/rcl_yaml_param_parser/COLCON_IGNORE; \
